### PR TITLE
Changes for demo setup.

### PR DESCRIPTION
**Pull request by slagle**

### CODE REVIEW

File: scripts/deploy-package-manifest.sh [hunk 1]
```diff
@@ -0,0 +1,47 @@
+set -x
```
Comment (dprince) [inline scripts/deploy-package-manifest.sh]: Could we name this scripts/deploy-catalogsource.sh? Really what we are creating here is the CatalogSource (and OperatorGroup). The Catalogsource is what drives OLM to deploy the packagemanifests for us. The script is the same just a simple rename.

File: pkg/operator/helpers.go [hunk 2]
```diff
@@ -288,7 +288,7 @@ func GetCSVBase(name, namespace, displayName, description, image, replaces strin
 			Namespace: "placeholder",
 			Annotations: map[string]string{
 				"alm-examples":   string(almExamples),
-				"capabilities":   "Basic Install",
+				"capabilities":   "Auto Pilot",
```
Comment (dprince) [inline pkg/operator/helpers.go:292]: Could we add a simple comment here too so we know to revert it later?

**Comment by dprince** [comment]
Two simple suggestions. Add a comment to the one file, and then could we just name the script something "catalogsource" as that is really the resource we directly create.

**Comment by slagle** [comment]
i got rid of the script changes. It's just the capability update now.

**Comment by slagle** [comment]
This PR is just changing the capability now, and I added the comment. Going to go ahead and merge this one.

**Pull request merged**